### ISSUE TITLE
Deprecation warning from npm install?

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#311]
Reported by @AmeliaBR https://twitter.com/AmeliasBrain/status/1064604479494983680

**Comment by AmeliaBR** [comment]
Specifically, it was that `postinstall-build` was deprecated and that npm native postinstall features should be used instead. Not sure if that was something you're using directly or if the warning was coming from a dependency.

**Comment by jevets** [comment]
```
warning @11ty/eleventy > nunjucks > postinstall-build@5.0.3: postinstall-build's behavior is now built into npm! You should migrate off of postinstall-build and use the new `prepare` lifecycle script with npm 5.0.0 or greater.
```

Looks like it's from nunjucks, just installed on node v10.13.0

**Comment by zachleat** [comment]
Filed upstream: https://github.com/mozilla/nunjucks/issues/1167

**Comment by zachleat** [comment]
Waiting on https://github.com/mozilla/nunjucks/pull/1172

**Comment by zachleat** [comment]
This will be included with 0.7.2
